### DISCUSSION TITLE
Removal of $@

### DISCUSSION THREAD

**Discussion by gertab** [tom5760/ezproxy-db#69] [General]
Currently urls are inserted in place of the` $@` symbol. E.g. 
> `https://ezproxy.beloit.edu/login?url=$@`

Given that the `$@` symbol is always found at the end of **all** proxy URLs (i.e. all proxy URLs prepend other URLs), is it really necessary to keep using the `$@` symbol?

**Comment by tom5760** [comment]
I think it's a good idea, but I guess it's currently working now, and gives a bit more flexibility.  One real reason to not change it, is that there are plugins using the database that I don't control that might break if we changed it.  I think the potential pain of breaking plugins expectations might not be worth the simplification.

**Reply by gertab** [reply]
> I think the potential pain of breaking plugins expectations might not be worth the simplification.

You're right, it will definitely cause compatibility issues with existing plugin.
